tests/ScenarioStateTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/*
+ * This file is part of the ScenarioStateBehatExtension project.
+ *
+ * (c) Rodrigue Villetard <rodrigue.villetard@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace Gorghoa\ScenarioStateBehatExtension;
+
+use Gorghoa\ScenarioStateBehatExtension\ScenarioState\Exception\MissingStateException;
+
+/**
+ * Class ScenarioStateTest
+ * @package Gorghoa\ScenarioStateBehatExtension
```

Comment:
Useless `@package` annotation
Can you add a `@author` annotation with your name please?